Source/Fuse.Controls.ScrollView/ScrollView.uno

```diff
@@ -377,7 +377,12 @@ namespace Fuse.Controls
 		*/
 		public void GotoRelative( float2 position )
 		{
-			Goto( (MaxScroll - MinScroll) * position + MinScroll );
+			Goto( RelativeToAbsolutePosition(position) );
+		}
+		
+		public float2 RelativeToAbsolutePosition( float2 pos )
```

Comment:
Is this really a good name? "Absolute position" can have multiple meanings, for instance "not relative to the position of the element", which isn't what this deals with... Maybe we should make this function internal to avoid making this a part of our public API?